server/engine/src/main/antlr4/org/eclipse/lsp/cics/parser/CICSParser.g4

```diff
@@ -232,7 +232,9 @@ cics_allocate_partner: PARTNER cics_name (NOQUEUE | STATE cics_cvda | cics_handl
 cics_asktime: ASKTIME cics_handle_response? (ABSTIME (cics_data_area | cics_handle_response)+)?;
 
 /** ASSIGN */
-cics_assign: ASSIGN (ABCODE cics_data_area | ABDUMP cics_data_area | ABPROGRAM cics_data_area | ACTIVITY cics_data_area |
+cics_assign: ASSIGN (cics_assign_parameter1 | cics_assign_parameter2) *;
+
+cics_assign_parameter1: ABCODE cics_data_area | ABDUMP cics_data_area | ABPROGRAM cics_data_area | ACTIVITY cics_data_area |
```

Comment:
what is the reason for having two rules here?

Reply:
This was just to mimic the grammar flow from the documentation https://www.ibm.com/docs/en/cics-ts/6.x?topic=summary-assign